ansible_collections/arista/cvp/plugins/module_utils/resources/schemas/v3.py

```diff
@@ -192,16 +192,13 @@
                     },
                     "imageBundle": {
                         "$id": "#/items/anyOf/0/properties/imageBundle",
-                        "type": "array",
-                        "title": "The imageBundle schema",
+                        "type": "string",
+                        "title": "The imageBundle name",
                         "description": "An explanation about the purpose of this instance.",
```

Comment:
This looks like a generic description which should be updated or removed.

Reply:
yeah - updated now